src/components/Header/index.js

```diff
@@ -29,9 +28,21 @@ const AuthenticatedHeaderOption = () => {
     });
   }, []);
 
+  const partnerGroupId = user.data.user.partner_group_id;
+
+  const canSpecifyPartnerGroupId =
+    (rolesList.includes("admin") ||
+      rolesList.includes("partner") ||
+      rolesList.includes("partner_view")) &&
```

Comment:
After importing the function:
```suggestion
    hasOneFrom(rolesList, ["admin", "partner", "partner_view"]) &&
```